### PR TITLE
refact: change login page to htmx and alpinejs and add setup page

### PR DESCRIPTION
**Pull request by polillomm**

### CODE REVIEW

**Review by polillomm — COMMENTED** [review]
Dev review ok.

**Comment by polillomm** [comment]
### Login page
image

### Setup page
image

File: src/presentation/ui/layout/setup.templ [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+					await Infinite.JsonAjax("POST", "/api/v1/setup/", {
+						username: this.username,
+						password: this.password,
+					});
+
+					const authResponse = await Infinite.JsonAjax("POST", "/api/v1/auth/login/", {
+						username: this.username,
+						password: this.password,
+					});
+
```
Comment (ntorga) [inline src/presentation/ui/layout/setup.templ:26]: I'd have check if tokenStr is a thing before trusting in it.

**Review by ntorga — CHANGES_REQUESTED** [review]

**Comment by polillomm** [comment]
### Login page
image

### Setup page
image

**Review by polillomm — COMMENTED** [review]
Dev review ok.

**Pull request merged**